apps/web/src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,9 @@
+import { Card, Flex } from "@chakra-ui/react";
+
+export const Sidebar = () => (
+  <Card gap="20px" padding="8px 8px 8px 20px">
+    <Flex gap="40px">
```

Comment:
Could it be just `<Flex gap="40px"/>`?

Reply:
I'm not sure this makes any sense since this component isn't finished yet and will be implemented later